index.js

```diff
@@ -175,6 +184,15 @@ alexa.response = function(session) {
   };
 };
 
+alexa.directives = function(directives) {
+  // load the alexa response directives information into details
+  this.details = directives;
+
+  this.set = function(directive) {
+    this.details.push(directive);
```

Comment:
@ajcrites This operates under the assumption that one wants to modify the mutable `self.response.response.directives` (initialized [here](https://github.com/ajcrites/alexa-app/blob/9eafffc493978208201b4d3fbe9ccf529dd985d1/index.js#L18)) rather than a passed-around object. Since this is only the second time I've played with Node, I'd love any input or feedback!